rice.gemspec

```diff
@@ -39,49 +35,19 @@ of the Ruby C API.
     'doxygen.ac',
     'doxygen.am',
 
-    'extconf.rb',
-
-    # Autoconf
-    'bootstrap',
-    'ax_cxx_compile_stdcxx.m4',
-    'configure.ac',
-    'configure',
-    'config.guess',
-    'config.sub',
-    'depcomp',
-    'doxygen.ac',
-    'doxygen.am',
-    'install-sh',
-    'missing',
-    'post-autoconf.rb',
-    'post-automake.rb',
-    'ruby.ac',
-    'aclocal.m4',
-
-    # Makefiles
+    # Other files
+    'Gemfile',
     'Rakefile',
-    'Makefile.am',
-    'Makefile.in',
-    'rice/Makefile.am',
-    'rice/Makefile.in',
-    'ruby/Makefile.am',
-    'ruby/Makefile.in',
-    'ruby/lib/Makefile.am',
-    'ruby/lib/Makefile.in',
-    'sample/Makefile.am',
-    'sample/Makefile.in',
-    'test/Makefile.am',
-    'test/Makefile.in',
 
-    # C++ source files
+    # rice.hpp
+    'include/rice.hpp',
```

Comment:
This probably should be `include/rice/rice.hpp`.

Reply:
Yup. I had started with #include <rice.hpp> and then decided #include <rice/rice.hpp> was a better idea for any future changes additions (like adding stl support or a backward compatibility header).